kde-apps/okteta/okteta-0.26.7.recipe

```diff
@@ -0,0 +1,233 @@
+	shared_mime_info$secondaryArchSuffix
+	devel:libKF5Attica$secondaryArchSuffix
+	devel:libKF5Auth$secondaryArchSuffix
+	devel:libKF5Bookmarks$secondaryArchSuffix
+	devel:libKF5Codecs$secondaryArchSuffix
+	devel:libKF5Completion$secondaryArchSuffix
+	devel:libKF5ConfigCore$secondaryArchSuffix
+	devel:libKF5ConfigWidgets$secondaryArchSuffix
+	devel:libKF5CoreAddons$secondaryArchSuffix
+	devel:libKF5Crash$secondaryArchSuffix
+	devel:libKF5DBusAddons$secondaryArchSuffix
+	devel:libKF5DocTools$secondaryArchSuffix
+	devel:libKF5I18n$secondaryArchSuffix
+	devel:libKF5IconThemes$secondaryArchSuffix
+	devel:libKF5ItemViews$secondaryArchSuffix
+	devel:libKF5JobWidgets$secondaryArchSuffix
+	devel:libKF5KCMUtils$secondaryArchSuffix
+	devel:libKF5KIOCore$secondaryArchSuffix
+	devel:libKF5NewStuff$secondaryArchSuffix
+	devel:libKF5NewStuffCore$secondaryArchSuffix
+	devel:libKF5Parts$secondaryArchSuffix
+	devel:libKF5Service$secondaryArchSuffix
+	devel:libKF5Solid$secondaryArchSuffix
+	devel:libKF5SonnetUi$secondaryArchSuffix
+	devel:libKF5TextWidgets$secondaryArchSuffix
+	devel:libKF5WidgetsAddons$secondaryArchSuffix
+	devel:libKF5WindowSystem$secondaryArchSuffix
+	devel:libKF5XmlGui$secondaryArchSuffix
+	devel:libqca$secondaryArchSuffix >= 2
+	devel:libQt5Core$secondaryArchSuffix >= 5.15
+	devel:libQt5DBus$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Qml$secondaryArchSuffix
+	devel:libQt5Script$secondaryArchSuffix
+	devel:libQt5ScriptTools$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	okteta${secondaryArchSuffix}_devel = $portVersion
+	devel:libKasten4Controllers$secondaryArchSuffix = 0.4.1
+	devel:libKasten4Core$secondaryArchSuffix = 0.4.1
+	devel:libKasten4Gui$secondaryArchSuffix = 0.4.1
+	devel:libKasten4Okteta2Controllers$secondaryArchSuffix = 0.4.0
+	devel:libKasten4Okteta2Core$secondaryArchSuffix = 0.4.0
+	devel:libKasten4Okteta2Gui$secondaryArchSuffix = 0.4.0
+	devel:libOkteta3Core$secondaryArchSuffix = 0.10.0
+	devel:libOkteta3Gui$secondaryArchSuffix = 0.10.0
+	"
+REQUIRES_devel="
+	okteta$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:gettext
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qdbuscpp2xml$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	# disable docs generation
+	sed -e '/add_subdirectory( doc )/ s/^#*/#/' -i CMakeLists.txt
+	sed -e '/kdoctools_install(po)/ s/^#*/#/' -i CMakeLists.txt
+
+	mkdir -p build
+	cd build
+
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$binDir \
+		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
+		-DCMAKE_INSTALL_DATAROOTDIR=$dataDir \
+		-DCMAKE_INSTALL_LIBDIR=$libDir \
+		-DCMAKE_INSTALL_BINDIR=$binDir \
+		-DSHARE_INSTALL_PREFIX=$dataDir \
```

Comment:
These could be combined (not tested yet)
```diff
	-DCMAKE_BUILD_TYPE=Release \
-		-DCMAKE_INSTALL_PREFIX=$binDir \
-		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
-		-DCMAKE_INSTALL_DATAROOTDIR=$dataDir \
-		-DCMAKE_INSTALL_LIBDIR=$libDir \
-		-DCMAKE_INSTALL_BINDIR=$binDir \
+		$cmakeDirArgs \
		-DSHARE_INSTALL_PREFIX=$dataDir \
```

Reply:
Didn't work unfortunately